test/ui-automation/test/specs/vault-interaction-flow.js

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+"use strict";
+
+import {wallet} from "../helpers";
+
+describe("TrustBloc Wallet - Add/Rename/Delete Vault flow", () => {
+    const ctx = {
+        email: `ui-aut-${new Date().getTime()}@test.com`,
+    };
+    const vaultEndpoint =  'vaults';
+    const validVaultName = 'Test Vault 1';
+    const invalidVaultName ='Vault!@';
+
+    // runs once before the first test in this block
+    before(async () => {
+        await browser.reloadSession();
+        await browser.maximizeWindow();
+    });
+
+    beforeEach(function () {});
+
+    it(`User Sign up (${ctx.email})`, async function () {
+        this.timeout(90000);
+
+        // 1. Navigate to Wallet Website
+        await browser.navigateTo(browser.config.walletURL);
+
+        // 2. Initialize Wallet (register/sign-up/etc.)
+        await wallet.init(ctx);
+
+        // 3. Go to Vaults screen
+        const vaultsLink = await $("#navbar-link-vaults");
+        await vaultsLink.click();
+    });
+    it(`User successfully adds vault`, async function () {
+        this.timeout(300000);
+        await wallet.addNewVault();
+        await wallet.vaultNameInput(validVaultName);
+        await wallet.createVault();
```

Comment:
Would it be possible to combine these 3 actions into one helper function ? In essence, this is adding a new vault with a name.

Reply:
done